src/Components/Buttons/CopyToClipboardButton.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useRef } from 'react';
+
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { t } from '@grafana/i18n';
+import { IconButton, InlineToast, useStyles2 } from '@grafana/ui';
+
+const SHOW_SUCCESS_DURATION = 2 * 1000;
+
+export default function CopyToClipboardButton({
```

Comment:
Does this one deprecate https://github.com/grafana/logs-drilldown/blob/main/src/Components/ServiceScene/CopyLinkButton.tsx this other?

Reply:
It very well could! Created: https://github.com/grafana/logs-drilldown/issues/1395